hibernate-core/src/main/java/org/hibernate/procedure/internal/ProcedureCallImpl.java

```diff
@@ -652,10 +652,11 @@ private ProcedureOutputsImpl buildOutputs() {
 		}
 
 		LOG.debugf( "Preparing procedure call : %s", call);
+		String sqlString = call.getSqlString();
```

Comment:
In theory we could store this in an instance variable and return it in `#getQueryString()`

Reply:
In the previous pull request I suggested passing it to the constructor of `ProcedureOutputsImpl` and `OutputsImpl`. Is that what you mean?

Anyway, as I said over there, I don't like seeing it passed to `executeStatement()`.

Reply:
What I mean is that `org.hibernate.procedure.internal.ProcedureCallImpl#getQueryString()` currently always returns `null`

https://github.com/hibernate/hibernate-orm/blob/ce48a51eaa50f14bbf81ee86ddb988107803aafb/hibernate-core/src/main/java/org/hibernate/procedure/internal/ProcedureCallImpl.java#L731

Instead of storing the query string in a local variable here we could also store it in an instance variable and additionally return it in `#getQueryString()`. I don't know whether this makes a difference somewhere. The passing to `OutputsImpl` could not be affected by this.

Reply:
That sounds reasonable.

Reply:
Do it here, but perhaps in a separate commit.